1209/DF00/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: mi:muz:tuch
+owner: D.F.Mac.@TripArts Music
```

Comment:
The 'owner' field needs to match the directory name of your org - in this case, 'TripArts-Music'.